docker-compose.yml

```diff
@@ -93,8 +93,10 @@ services:
       CONFIG_SERVICE_URL: http://core:8080
       ADAPTER_SERVICE_URL: http://adapter:8080
       TRANSFORMATION_SERVICE_URL: http://transformation:8080
-      NOTIFICATION_SERVICE_URL: http://notification:8080
       STORAGE_SERVICE_URL: http://storage:3000
+      AMQP_URL: amqp://rabbit_adm:R4bb!7_4DM_p4SS@rabbitmq:5672
+      AMQP_EXCHANGE: ods_global
+      AMQP_NOTIFICATION_EXECUTION_TOPIC: notification.execution.*
```

Comment:
I think for publishing you will need the exact topic.
Additionally, I would call the topic according to what happened, like "pipeline.execution.success"